README.md

```diff
@@ -70,18 +70,19 @@ $ yarn outdated-notifier --help
     $ outdated-notifier [options]
 
   OPTIONS:
-    --api-token       API token for GitHub (required)
-    --owner,      -o  repository owner name (default: "git config --get remote.origin.url" infomation)
-    --repository, -r  repository name (default: "git config --get remote.origin.url" infomation)
-    --title,      -t  issue title for GitHub
-    --labels,     -l  issue labels name for GitHub
-    --assignees,  -a  issue assignees name for GitHub
-    --excludes,   -e  path to yaml file which specify package names to exclude
-    --changelogs, -c  path to yaml file which specify changelog uris for the packages
-    --template        path to the template to use for notification
-    --dry-run         do not register on issue, output contents to stdout
-    --help,       -h  show help
-    --version,    -v  print the version
+    --api-token         API token for GitHub (required)
+    --githubApiAddress, provide address for github enterprise (default: api.github.com)
```

Comment:
Looks like good for me.
but, I would like you to make the change as follows :pray:

```suggestion
    --github-api-url, provide url for github enterprise (default: "api.github.com")
```

- Flag is unified with kebab-case.
- Rename to `url` from `address`. (IMO)

Reply:
no problem, I've made those changes now